liberty-maven-plugin/src/main/resources/net/wasdev/wlp/maven/plugins/MvnMessages.properties

```diff
@@ -322,3 +322,14 @@ error.install.dependency.app.missing=CWWKM2180E: The application dependency for
 error.install.dependency.app.missing.explanation=The application dependency cannot be found in Maven local repository.
 error.install.dependency.app.missing.useraction=Rebuild the project by running mvn install again.
 
+info.install.license=CWWKM2181I: Installing Liberty production license {0}.
+info.install.license.explanation=Installing Liberty production license to upgrade the Liberty runtime installation to the new edition and license terms.
+info.install.license.useraction=No action is required.
+
+info.install.license=CWWKM2182W: Liberty production license jar file {0} may be corrupted.
+info.install.license.explanation=The license file wlp/lafiles/LI_en is not found in the license jar file. The license jar file may be corrupted.
+info.install.license.useraction=Download and reinstall the Liberty license jar file to your local repository or internal custom repository.
+
+error.install.license=CWWKM2183E: Failed to install Liberty production license {0}. Return code is {1}.
+error.install.license.explanation=Failed to install Liberty production license. The edition and license terms for the current Liberty runtime installation stays the same.
```

Comment:
the explanation says: "The edition and license terms for the current Liberty runtime installation stays the same."  Since this is an error and causes an exception to be thrown, won't the whole install fail?  If we were going to proceed and let the install continue, I think it should be a warning, but if this ends processing, maybe this should give a bit more information on why the license install might have failed instead of saying the license and terms will stay the same.